### ISSUE TITLE
Add stream listener

### ISSUE THREAD

**Issue by blavejr** [CruiseDevice/twweet-cli#21]
Hi guys I took a look at your project and it's pretty interesting, I would like to help as well.
I was thinking why dont we add a stream listener so the user can get a constant stream of the latest tweets like on the twitter website or app without worrying about rate limits or downloading the same data over and over again.

I have started adding the stream listener to the project in my forked version, please take a look at it.

**Comment by CruiseDevice** [comment]
This is actually a good idea. @blavejr send a pr when you are done.

**Comment by 0x006E** [comment]
Hey @blavejr when sending PR remember to remove your Oauth details from the python file. I just your main py file and it was there. And also remove it when your pushing to github. It may compromise your account. Gd lck anyway on your fork :)

**Comment by CruiseDevice** [comment]
Something related to this i found here https://marcobonzanini.com/2015/03/02/mining-twitter-data-with-python-part-1/.

**Comment by CruiseDevice** [comment]
Also always try to be in sync with the main repository. May be this may happen, there have been some changes in the main repo after your fork. These changes does not automatically sync with your fork. You need to sync it manually.

**Comment by blavejr** [comment]
@CruiseDevice thanks for the heads up, I am new to open source and community projects so I still have a lot to learn. I have added the stream listener in my fork, it works pretty well. I just need a way to enable a keyboard interrupt so the user may escape from the stream and use other functionality of the client.

Thanks for the heads up on updating my fork I didn't know I had to do that, I will keep upto date.
Any other ideas that we might want to add to the client?
